pkg/networkmanager/network-interface-members.jsx

```diff
@@ -67,7 +67,7 @@ export const NetworkInterfaceMembers = ({
                     aria-label={cockpit.format(_("Switch of $0"), iface.Name)}
                     isDisabled={!privileged}
                     isChecked={!!(dev && dev.ActiveConnection)}
-                    onChange={val => {
+                    onChange={(_event, val) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230520-070544-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)